changelog/v1.19.0-beta3/7505-gw-api-cipher-suites.yaml

```diff
@@ -0,0 +1,9 @@
+
+changelog:
+- type: NEW_FEATURE
+  issueLink: https://github.com/solo-io/solo-projects/issues/7505
```

Comment:
Can we add https://github.com/k8sgateway/k8sgateway/issues/10169 and https://github.com/k8sgateway/k8sgateway/issues/10418 as well

Reply:
But those won't be solved in that repo? Not sure how the issue mirroring works

Reply:
I don't know... I'm also confused by this. My last issue in that repo we closed when it was resolved in gloo repos but it was a CI thing, so different. @nfuden?